data/shared/citizen/scripting/v8/main.js

```diff
@@ -15,9 +15,10 @@ const EXT_LOCALFUNCREF = 11;
 
 	const pack = data => msgpack.encode(data, { codec });
 	const unpack = data => msgpack.decode(data, { codec });
-	
+
 	// store for use by natives.js
 	global.msgpack_unpack = unpack;
+	global.msgpack_pack = pack;
```

Comment:
Why you need this in global?

Reply:
It's needed for packing function references among other things...